examples/huggingface_dataset.py

```diff
@@ -0,0 +1,252 @@
+# This is an example to apply PyTorch Frame tabular learning
```

Comment:
It's good to verify the data but I don't think this example is needed. We do not want to overflow our `examples/` with many duplicated examples with only datasets being changed.